pkg/uploader/provider/provider.go

```diff
@@ -40,7 +45,7 @@ type Provider interface {
 		path string,
 		tags map[string]string,
 		parentSnapshot string,
-		updater uploader.ProgressUpdater) (string, error)
+		updater uploader.ProgressUpdater) (string, bool, error)
```

Comment:
Please update the comment for the new returning value

Reply:
I've added comments